README.md

```diff
@@ -1,2 +1,20 @@
-# team
-Docs &amp; Assets Related To The MozMeao Team
+# MozMeao
+
+## Core Values
+
+* We treat each other, co-workers, and our community with respect, trust, and honesty.
+* We strive to make every project a public resource, open and accessible to all.
+* We bring passion, fun, and creativity to our work. 
+* We ship often, we ship high quality.
+
+
+## Our Purpose 
+
+Our team provides technical expertise and development resources to Marketing.
+
+
+## Our Mission 
+
+Leverage technology to create and support open, accessible, forward thinking products and services 
```

Comment:
Missing "and" after oxford comma.

Reply:
Nit: I would suggest a hyphen between `forward` and `thinking`, so `forward-thinking`.